ystia/yorc-a4c-plugin
# Get doc version from pom file

## Description of the change
Remove hard-coded version reference in documentation
Add python script in conf.py run by Sphinx to retrieve version from pom file

### How to verify it
Check the version value in documentation in index, install and location pages in readthedocs site.
